### PR TITLE
Moved Activity State

### PR DESCRIPTION
**Pull request by mbouadeus**
We moved the state that manages the activity type the user selected to a higher component for better accessibility throughout the app.

Please disregard commits:
Change access modifiers.
Complete schedule activity page UI.
Add state to schedule activty page.
Add title field for event.
Merge branch 'master' of https://github.com/googleinterns/step170-2020 … …
Merge branch 'master' of https://github.com/googleinterns/step170-2020 … …
Merge branch 'mvp-dev' of https://github.com/googleinterns/step170-2020… …
Moved login state to app component.
Minor

They are handled in different pull requests.
By the way, there might be some debug code since we weren't expecting to make these commits a pull request. They will be removed in a more up-to-date pull request.

### CODE REVIEW

**Review by trinitydonohugh — APPROVED** [review]

File: src/main/app/src/Routes.js [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
-export default Routes;
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import HomePage from './pages/HomePage';
+import ScheduleActivityPage from './pages/ScheduleActivityPage';
+import BrowsePage from './pages/BrowsePage';
+import HelpPage from './pages/HelpPage';
+import AboutPage from './pages/AboutPage';
+/* Routes address bar to corresponding page components */
```
Comment (pengxianzhe) [inline src/main/app/src/Routes.js]: nit: Add a empty line between imports and code/javadoc.

File: src/main/app/src/App.js [hunk 2]
```diff
@@ -7,18 +7,36 @@ import ResourceCard from './constants/ResourceCard';
...
+  const [isLoggedIn, updateIsLoggedIn] = React.useState(false);
+  const [accessToken, updateAccessToken] = React.useState("");
+  const [userId, updateUserId] = React.useState("");
+  const [activity, updateActivity] = React.useState("games");
+  const [links, updateLinks] = React.useState([]);
+
+  React.useEffect(() => {
+    updateActivityLinks(updateLinks,
+      activity == "active" ? videoData : activity == "reading" ? articleData : gameData);
+    console.log(links);
```
Comment (pengxianzhe) [inline src/main/app/src/App.js]: Remove test code.

File: src/main/app/src/constants/Navbar.js [hunk 3]
```diff
@@ -5,26 +5,31 @@ import info from './keys.js';
...
-  const clientID = info[0].clientID;
+  const clientID = "298755462-7fnqhho2db1b0mb5i6o7irgg8v7om76g.apps.googleusercontent.com";
   const discoveryDocs = "https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest";
   const scope = "https://www.googleapis.com/auth/calendar";
 
   // Handle successful login
   const handleLogin = (res) => {
     updateIsLoggedIn(true);
     updateGreeting("Welcome " + res.Ot.Cd +"!");
+    console.log(res);
```
Comment (pengxianzhe) [inline src/main/app/src/constants/Navbar.js]: Remove test code.

File: src/main/app/src/pages/HomePage.js [hunk 4]
```diff
@@ -5,16 +5,18 @@ import 'bulma/css/bulma.css';
...
   // Create state for activity selection with Games as default
-  const [activity, updateActivity] = React.useState("games");
+  // const [activity, updateActivity] = React.useState("games");
 
   // Update activty selection state based on dropdown
   const handleActivitySelection = evt => {
     updateActivity(evt.target.value);
   }
 
+  console.log(activity);
```
Comment (pengxianzhe) [inline src/main/app/src/pages/HomePage.js]: Remove test code.

File: src/main/app/src/pages/HomePage.js [hunk 5]
```diff
@@ -5,16 +5,18 @@ import 'bulma/css/bulma.css';
 import '../css/home.css';
 
 /* Component for home page */
-const HomePage = () => {
+const HomePage = ({activity, updateActivity}) => {
 
   // Create state for activity selection with Games as default
-  const [activity, updateActivity] = React.useState("games");
+  // const [activity, updateActivity] = React.useState("games");
```
Comment (isaacwdavis) [inline src/main/app/src/pages/HomePage.js]: Remove commented out code.

**Review by isaacwdavis — APPROVED** [review]

File: src/main/app/src/Routes.js (hunk 1 above)
Comment (mbouadeus) [reply src/main/app/src/Routes.js]: Fixed it.

File: src/main/app/src/App.js (hunk 2 above)
Comment (mbouadeus) [reply src/main/app/src/App.js]: Fixed it.

File: src/main/app/src/pages/HomePage.js (hunk 4 above)
Comment (mbouadeus) [reply src/main/app/src/pages/HomePage.js]: Fixed it.

File: src/main/app/src/constants/Navbar.js (hunk 3 above)
Comment (mbouadeus) [reply src/main/app/src/constants/Navbar.js]: Fixed it.

File: src/main/app/src/pages/HomePage.js (hunk 5 above)
Comment (mbouadeus) [reply src/main/app/src/pages/HomePage.js]: Fixed it.

**Review by pengxianzhe — APPROVED** [review]

**Pull request merged**